src/meta/metainfo.go

```diff
@@ -129,6 +129,7 @@ type FuncInfo struct {
 	Flags        FuncFlags
 	ExitFlags    int // if function has exit/die/throw, then ExitFlags will be <> 0
 	Internal     bool
+	IsVariadic   bool
```

Comment:
I would combine this with `Flags`. It already has information about `IsDeprecated`, `IsPure`.
Or create separate flags for the types.

Reply:
Make sense

Reply:
Done